_includes/charts/tigera-operator/templates/tigera-operator/02-configmap-calicoctl-resources.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if eq .Values.kubernetesProvider "openshift" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: calicoctl-resources
+  namespace: tigera-operator
+data:
+  # To have resources created before Calico resources are created add an
+  # entry here and the contents of the resource under the entry.
+  # Multiple resources/entries can be added to this ConfigMap.
+  #
+  # If you need to remove a resource that was added to this ConfigMap
+  # you should remove it from here or else upon restart of the operator
```

Comment:
"restart of the operator" is an implementation detail. We should just say "or else it will be re-created".